server/http/restapi/marketplace.go

```diff
@@ -0,0 +1,145 @@
+package restapi
+
+import (
+	"net/http"
+	"sort"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+type MarketplaceApp struct {
+	Installed bool                     `json:"installed"`
+	Labels    []model.MarketplaceLabel `json:"labels,omitempty"`
+	Manifest  apps.Manifest            `json:"manifest"`
+}
+
+func (a *restapi) handleGetMarketplace(w http.ResponseWriter, req *http.Request, actingUserID string) {
```

Comment:
Just to make sure. We are checking the user is authorized, but then we don't check anything else. Therefore, in theory, any user could fetch this information. I don't think there is any "sensitive information" returned here, and I understand eventually we want to allow all users to access the marketplace, but I wanted to be 100% sure.

Reply:
Good question. The behaviour here mimics the one from the Marketplace endpoint on the MM server. I don't see any reason not to open the endpoint for all users.